core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -77,11 +82,15 @@
  *
  * <p>The API of this class is subject to change without notice.
  */
-public class SslSocketFactory {
-  private static final Logger logger = Logger.getLogger(SslSocketFactory.class.getName());
+public class CoreSocketFactory {
+  public static final String CLOUD_SQL_INSTANCE_PROPERTY = "cloudSqlInstance";
```

Comment:
Do these all need to be public?

Reply:
Yes - the property is used to the Postgres SF to deal with a backwards compatibility issue  cause by a change in the PGdriver API (it carries the arg under a different key forward to the new key).

The two file formats are passed into the connect method (since the Postgres format requires a suffix). I could create an overloaded method for each, but at some point we may want users to be able to specify a different address, so I thought I would just expose the defaults for now.

Reply:
OK. We can deal with this in followup PRs, but I'm worried about String parameters that are really more constrained.